DESCRIPTION

```diff
@@ -25,13 +25,19 @@ BugReports: https://github.com/cdcgov/cfa-gam-rt/issues
 Suggests:
     testthat (>= 3.0.0),
     pkgdown,
-    withr
+    withr,
+    usethis,
+    primarycensoreddist
 Config/testthat/edition: 3
 Encoding: UTF-8
+Remotes: github::epinowcast/primarycensoreddist
```

Comment:
you might want to pin to a release via the universe vs using git? Regardless I am hoping to hit CRAN with it ASAP

Reply:
Is this possible? From[ a quick skim](https://github.com/r-lib/remotes/issues/618) it looks like it's hard?

Reply:
additional repositories?

Reply:
https://github.com/CDCgov/cfa-gam-rt/pull/37/commits/8d73c04f92a8711d086e996ff0f35833462484c5